benchmarks/README.md

```diff
@@ -1,4 +1,19 @@
-These are the benchmarks used in testing the different WebSocket server implementations. I use WebSocket++ as clients, since that is the fastest client library that I know of, and since you can run multiple instances of the client, it shouldn't really matter what client you use as long as the server is being stressed fully.
+# uWebSockets Benchmarks
+
+## Building Instructions
+
+First, install dependencies:
+
+* `bench1` depends only uWebSockets.
+* `bench2`, `bench3` and `wsPP` also depends on:
+  - [WebSocket++](https://github.com/zaphoyd/websocketpp) which is header-only, you can run `make install` in its directory to install the headers to system include path
+  - [boost](http://www.boost.org/) which must be built and installed, you could follow the [instructions for Linux/Mac](http://www.boost.org/doc/libs/1_60_0/more/getting_started/unix-variants.html) or [instructions for Windows](http://www.boost.org/doc/libs/1_60_0/more/getting_started/windows.html)
```

Comment:
Yes. For most people with a reasonably modern distribution would enjoy the convenience that the package managers provides. I was in the context of my old distribution on production server and neglected to mention.